ikura/shim/proto/da.proto

```diff
@@ -0,0 +1,131 @@
+// taken as is from:
+// https://github.com/rollkit/go-da/blob/4538294d1704417d0b80273e6437ba1de5548e8a/da.go
+
+syntax = "proto3";
+package da;
+
+// DAService is the protobuf service definition for interaction with Data Availability layers.
+service DAService {
+    // MaxBlobSize returns the maximum blob size
+    rpc MaxBlobSize(MaxBlobSizeRequest) returns (MaxBlobSizeResponse) {}
```

Comment:
indent is off here compared to other lines

a few more examples in this file

Reply:
Thanks, will fix.

Reply:
I realized that it was actually verbatim copy and it had mixed whitespaces. I went ahead and converted it anyway, since it became not verbatim code after me adding the attribution/provenance notice at the top of the file:

https://github.com/thrumdev/blobs/blob/bc180253be6bba0eab7bce75e99dac2b973181a1/ikura/shim/proto/da.proto#L2